samples/subsys/mgmt/mcumgr/smp_svr/overlay-bt.conf

```diff
@@ -5,31 +5,31 @@ CONFIG_BT_BUF_ACL_TX_SIZE=502
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
 
 # Enable the Bluetooth mcumgr transport (unauthenticated).
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL=y
+CONFIG_MCUMGR_TRANSPORT_BT=y
```

Comment:
IMO this commit should be squashed into 1st commit, so that this patch series is bisectable.